src/lib/ownership/OwnershipEngine.ts

```diff
@@ -39,7 +39,7 @@ export class OwnershipEngine {
   }
 
 
-  public static FromCodeownersFile(filePath: string) {
+  public static FromCodeownersFile(filePath: string, allowRelativePaths: boolean) {
```

Comment:
i wonder if we don't add an argument, but just make it support this by default?

Reply:
Oh I think that will work for us, we can send that value to the `ignore` method in this file:
```
const match: any = ignore({ allowRelativePaths: true }).add(path);
```
I'll add it